### PR TITLE
fallback to event text in location body when map unavailable

### PR DESCRIPTION
**Pull request by kerryarchibald**
Fixes: https://github.com/vector-im/element-web/issues/20655

Includes https://github.com/matrix-org/matrix-react-sdk/pull/7967

https://github.com/matrix-org/matrix-react-sdk/compare/psf-683/send-pin-drop-events...psf-657/bad_map_style_url

`map_style_url` is not configured in .well-known or config.json
Screenshot 2022-03-04 at 16 15 35

`map_style_url` is set in .well-known or config.json but is unreachable
Screenshot 2022-03-04 at 16 13 48

Own location (pin drop share used above)
Screenshot 2022-03-04 at 16 18 51

Screenshot 2022-03-04 at 17 44 28
Screenshot 2022-03-04 at 17 42 30

---
Here's what your changelog entry will look like:

## ✨ Features
 * fallback to event text in location body when map unavailable ([\#7982](https://github.com/matrix-org/matrix-react-sdk/pull/7982)). Fixes vector-im/element-web#20655. Contributed by @kerryarchibald.

Preview: https://pr7982--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/location/findMapStyleUrl.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { logger } from "matrix-js-sdk/src/logger";
```
Comment (turt2live) [inline src/components/views/location/findMapStyleUrl.ts:1]: needs copyright

**Review by turt2live — APPROVED** [review]
generally seems fine!

**Review by gaelledel — CHANGES_REQUESTED** [review]
Could we use this style? https://www.figma.com/file/uthEK6xuo9hSQRdWvPAre5/Location-Sharing?node-id=793%3A52185

**Comment by kerryarchibald** [comment]
Screenshot 2022-03-09 at 18 51 15

**Review by gaelledel — APPROVED** [review]

**Pull request merged**